pages/learn/develop/integrations/aws.md

```diff
@@ -34,10 +34,17 @@ Each `device` or `thing`, must have its own certificates that will be used to au
 
 ![generate certs](/img/integrations/aws/04_create_cert.png)
 
-In order to authenticate with the service, you will first need to download all four certificates from the dashboard to your computer (make sure you also download the `root CA` for AWS IoT).
+In order to authenticate with the service, you will first need to download the following files
+
+1. The thing's certificate
```

Comment:
I haven't tried building locally, but will this list populate with the correct numbers @rahul-thakoor ?

Reply:
yes